src/vs/workbench/parts/tasks/electron-browser/terminalTaskSystem.ts

```diff
@@ -391,7 +391,7 @@ export class TerminalTaskSystem extends EventEmitter implements ITaskSystem {
 			} else {
 				(this.terminalService.configHelper as TerminalConfigHelper).mergeDefaultShellPathAndArgs(shellLaunchConfig);
 			}
-			let shellArgs = shellLaunchConfig.args.slice(0);
+			let shellArgs = <string[]>shellLaunchConfig.args.slice(0);
```

Comment:
@weinand FYI IShellLaunchConfig can now have args as a string to support @dbaeumer's use case of needed to use a pre-escaped set of arguments. I needed to make this change which I believe shouldn't cause any issues.